xbmc/interfaces/python/Doxyfile

```diff
@@ -2000,7 +2000,7 @@ INCLUDE_FILE_PATTERNS  =
 # recursively expanded use the := operator instead of the = operator.
 # This tag requires that the tag ENABLE_PREPROCESSING is set to YES.
 
-PREDEFINED             =
+PREDEFINED             = SWIG
```

Comment:
Yes, this was required to swap to the old documentation of functions.

Here the examples for:
```cpp
#ifndef SWIG /* Part below used for HTML doc and behind #else for python internal docs */
      ///
      /// \ingroup python_InfoTagMusic
      /// @brief Returns the artist from music as string if present.
      ///
      /// @return [string] Music artist
      ///
      ///
      ///------------------------------------------------------------------------
      ///
      ///
      String getArtist();
#else
      /**
       * getArtist() -- returns a string.\n
       */
      String getArtist();
#endif
```

The upper visible here on website and with `#ifndef SWIG` readed there from his doxygen:
http://alwinesch.github.io/a00300.html#ga1b9f565753d008491c59887680bd570e

And for the other selected with this `SWIG` from the other Doxyfile to create this on AddonModuleXbmc.i.cpp:
´´´cpp
  PyDoc_STRVAR(XBMCAddon_xbmc_InfoTagMusic_getArtist__doc__,
               "getArtist() -- returns a string.\n");

  static PyObject* xbmc_XBMCAddon_xbmc_InfoTagMusic_getArtist (PyHolder* self  , PyObject *args, PyObject *kwds  )
  {
    ...
´´´